packages/cmf/src/UIRouter.js

```diff
@@ -1,37 +1,125 @@
 /**
  * @module react-cmf/lib/UIRouter
  */
-import React from 'react';
-import { Router as BaseRouter } from 'react-router';
+import React, { PropTypes } from 'react';
+import { Route } from 'react-router';
+import { ConnectedRouter } from 'react-router-redux';
 import { connect } from 'react-redux';
-
+import { createHashHistory } from 'history';
 import api from './api';
 
+function CMFRoute(props, context) {
+	let Component = api.route.getComponentFromRegistry(context, props.component);
+	if (props.view && !Component.CMFContainer) {
+		Component = api.route.connectView(context, Component, props.view);
+	}
+	if (props.path === '/') {
```

Comment:
Since this is only for the root configuration, it would be better to move this directly in `CMFRouter`

Another thing that we mentionned together was to use the `<Redirect>` component from react-router instead of our custom one for the `/` route. This would introduce a breaking change, since settings.json would change for this root route